app/views/layouts/partials/_hero-basic.html.erb

```diff
@@ -1,5 +1,20 @@
-<header class="hero--basic">
+<% classes = '' if local_assigns[:classes].nil? %>
```

Comment:
Could the assignments on the html.erb files be transformed into helper functions instead please?